### PR TITLE
L10n japanese updates 2023 11 17

### PR DESCRIPTION
**Pull request by steppi**
#### Brief description of what is fixed or changed

Update Japanese translations. I saw on Crowdin that @AtsushiSakai translated the updated news content and adjusted some translations so making a PR to get these updates in.

Crowdin is completely out of sync with github because we didn't merge Crowdin's autogenerated PR, instead taking only the commits we wanted. In addition, translations on main also had some updates made directly without using Crowdin. My process here is to do a scripted rebase on the commit where l10n_main diverges from main which filters out all commits not from the language of interest, and squashes together commits that touch the same file (fortunately Crowdin doesn't make commits that touch multiple files). I then rebase this on main and manually clean up the merge conflicts.

@AtsushiSakai, please let me know if everything looks Ok. cc @tkoyama010 in case you want to take a look too.

### CODE REVIEW

File: content/ja/news.md [hunk 1]
```diff
@@ -1,35 +1,49 @@
...
+### numpy.orgが日本語とポルトガル語で利用可能になりました
+
+_2023年4月2日_ -- numpy.orgが2つの言語で利用可能になりました： 日本語とポルトガル語。 熱心なボランティアがいなければ、このプロジェクトは不可能でした：
+
+_ポルトガル語_
+* Melissa Weber Mendonça (melissawm)
 * Ricardo Prins (ricardoprins)
-* Getúlio Silva (getuliosilva)
+* Getúlio Silva (getuliosilva)
+>>>>>>> 4528610 (New translations news.md (Japanese))
```
Comment (jarrodmillman) [inline content/ja/news.md:31]: This looks like it should be removed.

File: content/ja/news.md [hunk 2]
```diff
@@ -212,6 +226,10 @@ _2019年11月15日_ -- NumPyと、NumPyの重要な依存ライブラリの1つ
 
 こちらは、より以前のNumPyリリースのリストで、各リリースノートへのリンクが記載されています。 全てのバグフィックスリリース(バージョン番号`x.y.z` の`z`だけが変更されたもの)は新しい機能追加はされず、マイナーリリース (`y` が増えたもの)は、新しい機能追加されています。
 
+<<<<<<< HEAD
```
Comment (jarrodmillman) [inline content/ja/news.md:229]: This too.

**Review by AtsushiSakai — APPROVED** [review]
LGTM except points @jarrodmillman pointed.

File: content/ja/news.md [hunk 3]
```diff
@@ -69,7 +83,7 @@ Numpy 1.25. リリースは引き続きdtypeの取り扱いと dtypeのプロモ
 
 ### Numpy 1.23.0 リリース
 
-_2022年1月22日_ -- [Numpy 1.23.0](https://numpy.org/doc/stable/release/1.23.0-notes.html) がリリースされました。 今回のリリースのハイライトは次のとおりです。
+_2022年1月22日_ -- [Numpy 1.23.0](https://numpy.org/doc/stable/release/1.23.0-notes.html) がリリースされました。 今回のリリースの目玉機能は次のとおりです。
```
Comment (tkoyama010) [inline content/ja/news.md:86]: Let's unify the expression with the other part.
```suggestion
_2022年1月22日_ -- [Numpy 1.23.0](https://numpy.org/doc/stable/release/1.23.0-notes.html) がリリースされました。 今回のリリースのハイライトは次のとおりです。
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: content/ja/news.md (hunk 1 above)
Comment (steppi) [reply content/ja/news.md]: Thanks. Not sure how I missed these merge artifacts.

**Comment by steppi** [comment]
Thanks @jarrodmillman for pointing out the merge artifacts, @AtsushiSakai for the review, and @tkoyama010 for the suggestions. I think this is ready now if someone wants to merge it.

**Pull request merged**